op-preimage/verifier_test.go

```diff
@@ -43,26 +45,26 @@ func TestWithVerification(t *testing.T) {
 			name:        "Keccak256 InvalidData",
 			key:         keccak256Key,
 			data:        []byte{6, 7, 8},
-			expectedErr: ErrIncorrectData,
+			expectedErr: preimage.ErrIncorrectData,
 		},
 		{
 			name:        "EmptyData",
 			key:         keccak256Key,
 			data:        []byte{},
-			expectedErr: ErrIncorrectData,
+			expectedErr: preimage.ErrIncorrectData,
 		},
 		{
 			name:        "UnknownKey",
 			key:         invalidKey([32]byte{0xaa}),
 			data:        []byte{},
-			expectedErr: ErrUnsupportedKeyType,
+			expectedErr: preimage.ErrUnsupportedKeyType,
 		},
 	}
 
 	for _, test := range tests {
-		test := test
+		//test := test
```

Comment:
Why did you comment this out? See https://github.com/golang/go/issues/60078. I believe the problem was fixed in Go 1.21 but we need to be very mindful about this change, would prefer if you uncomment this line

Reply:
Whoops, that was part of a different change I was working on, and I didn't meant to include that here. I've reverted this.

Sorry about this! I know it's super annoying when someone submits a PR with a bunch of repetitive non-functional changes and one change that changes behavior, and I didn't mean to do that here.

Reply:
Thank you for the thoughtfulness! I will resolve now